winrm4j/src/test/java/io/cloudsoft/winrm4j/winrm/WinRmToolExecLiveTest.java

```diff
@@ -75,7 +75,39 @@ public void testExecFailingScript() throws Exception {
     @Test(groups="Live")
     public void testExecScriptExit0() throws Exception {
         assertExecSucceeds("exit /B 0", "", "");
-        assertExecSucceeds(ImmutableList.of("exit /B 0"), "", "");
+    }
+
+    @Test(groups = "Live")
+    public void testChainCommands() {
+        assertExecCommand("echo Hi & echo World", "Hi \r\nWorld", "", 0);
+    }
+
+    /**
+     * Please bear in mind this behavior when executing commands.
+     */
+    @Test(groups="Live")
+    public void testExecRNSplitExit() throws Exception {
+        assertExecCommand("echo Hi\r\necho World\r\n", "Hi", "", 0);
```

Comment:
@aledsage I wouldn't say it should be a failing test.
This is the way it behaves and I think this is the way it should be.
Everybody advice to use ` & ` for chaining commands. `\r\n` is our fiction appeared initially in the Jython's version.